### DISCUSSION TITLE
Exception thrown at 0x6100284A (f_sps.dll) in BlankWinApp.exe

### DISCUSSION THREAD

**Discussion by kaki104** [microsoft/WindowsAppSDK#2002] [Q&A] [answered]
I have created and run a basic project, but I am getting an error.

Environment
Windows 11 Version 21H2 (OS Build 22000.434)
Visual Studio 2022 version 17.0.5

f_sps.pdb not loaded

f_sps.pdb contains the debug information required to find the source for the module f_sps.dll
Module Information
version : 2.08.3.20
original location : c:\program files (x86)\Fasoo DRM\f_sps.dll

ERROR
Exception thrown at 0x6100284A (f_sps.dll) in BlankWinApp.exe: 0xC0000005: Access violation reading location 0x00000000.

And, I can proceed by clicking the Continue button.
Even when I run it as a release, I get an error.

If DRM is installed, what should I do about the part where an error occurs when running the app? Should I keep ignoring the error?

**Comment by riverar** [comment, marked as answer]
I would suggest contacting Fasoo for support of Fasoo Enterprise DRM. Perhaps their component doesn't support Windows 11 or Windows App SDK? DRM software tends to be very specialized and very picky.
